charter.html

```diff
@@ -241,6 +241,17 @@ <h2 id=proposals>Proposals</h2>
 expressed interest in the Proposal
 </ul>
 
+<p>Proposals begin as or evolve into
+<a href=https://github.com/w3ctag/w3ctag.github.io/blob/master/explainers.md>explainers</a>
+which describe the proposed changes to the web platform, and which may
+serve as the basis for a Work Item (should the Proposal be adopted as
+one).
+
+<p class=note>While explainers are Community Group Reports as defined in
+the <a href=https://www.w3.org/community/about/agreements/>Community and
+Business Group Process</a>, they are not Specifications as defined in
+that document.
```

Comment:
So this makes me a bit nervous -- it means contributions to explainers aren't covered under the CLA.  Perhaps that's a bug in the community group process, but it feels like it would perhaps be better to go the other direction.

I guess it's better to be clear one way or the other rather than the current possibly-ambiguous situation... but I'm not especially happy about this.

Reply:
> So this makes me a bit nervous -- it means contributions to explainers aren't covered under the CLA.

Oof, yeah. The CBGP and CLA clearly only apply the CLA to Specifications, and not to other Community Group Reports or other Community Group deliverables (the catch-all category that includes all Community Group Reports as well as "test suites, tutorials, demos, code, discussion, etc.")

> Perhaps that's a bug in the community group process, but it feels like it would perhaps be better to go the other direction.
> 
> I guess it's better to be clear one way or the other rather than the current possibly-ambiguous situation... but I'm not especially happy about this.

Neither am I. In privacycg/privacycg.github.io#21 I propose text that requires the Chairs to "ensure contributions to Proposals are only made by Community Group Participants who have agreed to the <a href=https://www.w3.org/community/about/agreements/cla/>W3C Community Contributor License Agreement (CLA)</a>". Now that you've pointed this out, though, I'm not sure if that text even makes sense given the CBGP and CLA text.

Our *intent* when defining Proposals was either to put a name to (a) a "discussion" (a form of Community Group deliverable that is not a Community Group Report), or (b) a (non-Specification) Community Group Report. Either way, they don't get covered by the current terms of the CLA.

I think this is arguably a bug in the CBGP & CLA. Taking a look around at other CG charters, you see language like this from the [WICG charter](https://wicg.github.io/admin/charter.html):

> The Community Group will accept and discuss any proposal for a web platform feature that would be implemented in a browser or similar user agent. Any suggestions, pull requests, issues, or comments made about a proposal fall under the CLA.

This reads to me as if WICG is attempting to apply the CLA well beyond what the CBGP or CLA envision.

Or consider this language from the [CG charter template](https://w3c.github.io/cg-charter/CGCharter.html) that lots of CGs use:

> W3C will seek and expect an organizational commitment under the CLA starting with the individual's first request to make a contribution to a group Deliverable.

That makes it sound like CLA commitments happen for any Community Group deliverable, which include things like tutorials, demos, and discussion. But the CG charter template goes on to say the following:

> Substantive Contributions to Specifications can only be made by Community Group Participants who have agreed to the W3C Community Contributor License Agreement (CLA).

So, yeah. I think this is a mess, but the mess is not unique to the Privacy CG. It appears to be a mess across the board.

Reply:
@wseltzer probably has good input here.

Reply:
@dbaron I'm inclined to land this and #21, which at least puts us in a similar situation to all the other CGs. Could you separately raise the underlying concern on the CBGP itself?

Reply:
I've asked in the w3ccommunity Slack about where one would go about filing an issue on the CBGP.

Reply:
Filed w3c/cg-council#1 to capture this.